### DISCUSSION TITLE
Best sources for Upper Air data

### DISCUSSION THREAD

**Discussion by ccostanza10** [Unidata/siphon#656] [General]
In general, is it best to stick with using the Wyoming request to get Radiosonde data? I'm struggling to get data from specific radiosonde locations via this Wyoming request function like this example - https://unidata.github.io/siphon/latest/examples/upperair/Wyoming_Request.html

**Comment by dopplershift** [comment]
Wyoming is usually my first choice, then if that failed I'd use Iowa. What site is giving you problems?

**Comment by ccostanza10** [comment]
For instance, 01028 - http://weather.uwyo.edu/cgi-bin/bufrraob.py?datetime=2023-01-24%2012:00:00&id=01028&type=INVENTORY, looks like radiosonde data is available every 6 hours. I'm getting this error when I run the wyoming request through siphon.

ValueError: No data available for 2023-01-24 00Z for station 01028.

**Reply by dopplershift** [reply]
While the BUFR inventory shows it, I'm unable to bring it up using the access point we use:

https://weather.uwyo.edu/cgi-bin/sounding?region=np&TYPE=TEXT%3ALIST&YEAR=2023&MONTH=01&FROM=2412&TO=2412&STNM=01028

So not much Siphon can do about that. Not sure why there's a discrepency.

**Reply by ccostanza10** [reply]
Right, I think it gets at a larger issue that some European operational sites are now only sharing their data in BUFR format. Still, I actually directly reached out to Larry Oolman from Wyoming, and he was able to explain a few things. In short, Siphon won't work for this example. All good! Thanks looking into this.
